src/server.c

```diff
@@ -6524,10 +6524,7 @@ void setupSignalHandlers(void) {
 
     setupSigSegvHandler();
 
-    act.sa_flags = SA_SIGINFO;
-    act.sa_handler = NULL;
-    act.sa_sigaction = sigalrmSignalHandler;
-    sigaction(SIGALRM, &act, NULL);
+    setupSigAlrmHandler();
```

Comment:
maybe we should call just one `setupDebugSigHandlers` instead of both of these?
i.e. some init function for debug.c to do whatever it needs..

Reply:
We can have `setupDebugSigHandlers` that wraps `setupSigSegvHandler` and `setupSigAlrmHandler`
setupSigSegvHandler still has to be exposed since it is used by `updateSighandlerEnabled`

Reply:
ohh, i missed that. well, in that case, let's keep what you have now

Reply:
ohh, i see you changed it.. well, that's ok too.